server/migrations/users.migration.js

```diff
@@ -0,0 +1,61 @@
+export default {
+  up(queryInterface, Sequelize) {
+    return queryInterface.createTable("Users", {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      firstName: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      middleName: {
+        type: Sequelize.STRING
+      },
+      lastName: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true
+      },
+      gender: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      biography: {
+        type: Sequelize.TEXT
+      },
+      mobileNumber: {
+        type: Sequelize.CHAR
+      },
+      updatedAt: {
+        type: Sequelize.DATE
+      },
+      imageURL: {
+        type: Sequelize.TEXT
+      },
+      createdAt: {
```

Comment:
Please put the date fields at the bottom of the model as is in convention in most code bases. I am not seeing anything fantastic that you are doing with the timestamps fields that is making you disabled them. Please un-disable them